src/botl.c

```
@@ -989,15 +1189,28 @@ bot_via_windowport(void)
     Strcpy(nb = buf, svp.plname);
     nb[0] = highc(nb[0]);
     titl = !Upolyd ? rank() : pmname(&mons[u.umonnum], Ugender);
+#if 0 /*JP*/
     i = (int) (strlen(buf) + sizeof " the " + strlen(titl) - sizeof "");
+#else
+    i = (int) (strlen(buf) + sizeof " " + strlen(titl) - sizeof "");
```

Cppcheck (2.21.0)

[error] Uninitialized variable

(uninitvar)